source/guides/administration.rst

```diff
@@ -3,15 +3,15 @@ Mattermost administration guide
 
 This guide is for people who administer an existing Mattermost server. It’s divided into four parts:
 
-* `The basics <https://docs.mattermost.com/guides/administration.html#the-basics>`__ - An overview of standard configurations and features.
+* `Get started <https://docs.mattermost.com/guides/administration.html#get-started>`__ - An overview of standard configurations and features for both self-hosted and Cloud workspaces.
 * `Cloud workspace management <https://docs.mattermost.com/guides/administration.html#workspace-management>`__ - Detailed information about admin capabilities of Mattermost Cloud workspaces.
 * `Self-hosted administration <https://docs.mattermost.com/guides/administration.html#self-hosted-admin>`__ - Detailed information about admin capabilities of self-hosted Mattermost servers.
 * `Other resources <https://docs.mattermost.com/guides/administration.html#other-resources>`__ - Additional migration, conversion, and troubleshooting resources.
 
-If you’re looking for resources to help you install, deploy, and scale your self-hosted Mattermost server, refer to the :doc:`Mattermost Deployment Guide </guides/deployment>`.
+If you’re looking for resources to help you install, deploy, and scale your self-hosted Mattermost server, see the :doc:`Mattermost Deployment Guide </guides/deployment>`.
 
-The basics
-----------
+Get started
+------------
 .. toctree::
```

Comment:
Does this TOC still apply? I noticed it doesn't match the descriptions from line 39 onward

Reply:
Thanks, @jasonblais - it does still apply, and I agree that the LHS and the landing page need to be more parallel. It's important that users can reach individual pages of interest through both the landing page and the LHS on demand. Good catch!